### ISSUE TITLE
FY2023 Q2

### ISSUE THREAD

**Issue by chadwhitacre** [getsentry/team-ospo#33]
[old slide deck](https://docs.google.com/presentation/d/1dmCmFUKgN-nY42akQxkUdNl1eMOLWNCA6EoqNunuiqM/edit#slide=id.g11b425b447c_0_84) &middot; [Q3 →](https://github.com/getsentry/team-ospo/issues/34)

# Hire

- [x]  ~~Open Source Program Manager~~ role cut
- [x]  Software Engineer — pulled up from Q3, @ethanhs 💃 
- [x]  Software Engineer — pulled up from Q4, starts in a month
- [x] [Bad Selfies](https://www.notion.so/sentry/Open-Source-Program-Office-9d79a7d3f2a84ec986e463e17fe5635d#0b767b3372a940688765c2090b1bcfa7) 👍

# Maintain

- [x] Self-hosted
    - [x] [22.5.0](https://github.com/getsentry/self-hosted/releases/tag/22.5.0)
    - [x] [22.6.0](https://github.com/getsentry/self-hosted/releases/tag/22.6.0)
    - [x] [22.7.0](https://github.com/getsentry/self-hosted/releases/tag/22.7.0)
    - [x] @ethanhs is [productive](https://github.com/getsentry/self-hosted/issues?q=commenter%3Aethanhs+created%3A%3C%3D2022-07-27) in [his first 30 days](https://user-images.githubusercontent.com/134455/181383178-fcc25be3-1876-4692-8019-8ae9e52a4146.png)
- [x] TTR/TTT
    - [91% / 79% under SLO](https://sentryio.cloud.looker.com/dashboards/401)
    - steady for a year now
    - [It worked](https://user-images.githubusercontent.com/134455/181356636-f8411400-3c8d-4000-a5eb-a55e37260b65.png).
    - Support team is doing great.
- [x] FOSSA
    - This was rocky, we had to turn it off for a time while they fixed bugs.
    - It's too noisy but Legal keeps on top of it pretty responsively.
    - We never shipped attribution reports.

# Build

- [x] **Engagement**
    - Shipped [viz on 401](https://sentryio.cloud.looker.com/dashboards/401) and started weekly 🌮s—[one](https://sentry.slack.com/archives/C1B4LB39D/p1657049163803199), [two](https://sentry.slack.com/archives/C1B4LB39D/p1657744088356239), [three](https://sentry.slack.com/archives/C1B4LB39D/p1658253234073279), [four](https://sentry.slack.com/archives/C1B4LB39D/p1658858077704909)
        - [the pattern](https://user-images.githubusercontent.com/134455/181364902-7fbb69ef-fe0e-4440-8f5f-d4b08b0ad573.png) [is infectious](https://user-images.githubusercontent.com/134455/181384383-a8390118-6ff3-4ed7-9a5d-129bff13005d.png) and ([thanks Bruno](https://user-images.githubusercontent.com/134455/181364901-2a890133-4435-4b21-82ac-4ac90b95a87f.png)) [I'm prolific](https://user-images.githubusercontent.com/134455/181364905-eb727744-86e3-4c86-9655-575215b625f5.png)
    - Wrote a doc on open source's role in product dev called [Bump, Set, Spike](https://www.notion.so/sentry/Bump-Set-Spike-c714618f881b4579b80b703184ebeadb)
        - using this to track and amplify [small wins](https://www.notion.so/sentry/Bump-Set-Spike-c714618f881b4579b80b703184ebeadb#7c3e1fd4dca647b4884557e5628bd53e)
    - Also spiraled into ...
- [x] **Research EPD move to GitHub**
    - [Move EPD to GitHub?](https://www.notion.so/sentry/Move-EPD-to-GitHub-0382d12bc2ec4a44a8562191f31a69ff)
    - [What Happened to GitHub?](https://www.notion.so/sentry/What-happened-to-GitHub-7979d50330ea4f70b570c608a6931b2c)
    - tl;dr Data doesn't support a super-disruptive wholesale move.
    - I spent pretty much the whole quarter on this unexpectedly, and it kinda sucked but it would suck worse if we were still trying to move forward with it. Close one! 😰
- [x] ~~**Education**~~ dropped
